### PR TITLE
pybind/mgr/restful: use dict.items() for py3 compatible

### PR DESCRIPTION
**Pull request by tchaikov**
Signed-off-by: Kefu Chai <kchai@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2019-07-27_05:17:08-rados-wip-kefu-testing-2019-07-27-1103-distro-basic-smithi/
- http://pulpito.ceph.com/kchai-2019-07-27_09:39:02-rados-wip-kefu-testing-2019-07-27-1103-distro-basic-smithi/
- http://pulpito.ceph.com/kchai-2019-07-28_02:01:48-rados-wip-kefu-testing-2019-07-27-2234-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2019-07-28_11:06:54-rados-wip-kefu-testing-2019-07-27-2234-distro-basic-mira/

the last two failures should be addressed by #29368

File: src/pybind/mgr/restful/common.py [hunk 1]
```diff
@@ -34,7 +36,7 @@
 def humanify_command(command):
     out = [command['prefix']]
 
-    for arg, val in command.iteritems():
+    for arg, val in six.iteritems(command):
```
Comment (sebastian-philipp) [inline src/pybind/mgr/restful/common.py]: if this is not critical, we might just use

```suggestion
    for arg, val in command.items():
```

instead.

Comment (tchaikov) [reply src/pybind/mgr/restful/common.py]: i am fine with either of these two approaches as long as we don't care about creating a copy of `command` in python2.

**Comment by tchaikov** [comment]
@sebastian-philipp updated and repushed.

**Comment by sebastian-philipp** [comment]
(still approve)

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2019-07-30_11:47:18-rados-wip-sage-testing-2019-07-29-1938-distro-basic-smithi/

**Pull request merged**